src/hooks/use-element-should-close.js

```diff
@@ -63,7 +63,9 @@ export function useElementShouldClose(closeableEl, isOpen, handleClose) {
       document.body,
       ['focus'],
       event => {
-        if (!closeableEl.current.contains(/** @type {Node} */ (event.target))) {
+        if (
+          !closeableEl.current?.contains(/** @type {Node} */ (event.target))
```

Comment:
I think this is correct: if `closeableEl.current` is `null` we want to call `handleClose` function (doesn't matter where the focus event originated from). However, when the `closeable.current` element is defined the focus event should originate from  within.

Is that correct?

Reply:
There is a change in the line, a `?` is added. 

Because `closeableEl` is a mutable ref, the value of `current` could be `null`. I was asking you to double check the logic when that is the case. I think it works well.

Reply:
Good catch. I think this could cause unexpected behavior. I'd like to take a closer look.